Part3_InterruptRoutingSetupInAChipset.html

```diff
@@ -56,19 +56,19 @@ <h3>Why we just can't connect INTA#→PIRQA, INTB#→PIRQB, ... everywhere?</h3>
 		<li>INTD#→PIRQD</li>
 	</ul>
 
-	<p>As we've already said above, the most common case is when a PCI device has only one function, and its interrupt is connected to the INTA# lane (because why whould hardware device developer route it in a different way?). Therefore if we would decide to route all lanes as we've written, almost all of the devices in a system would share interrupt lane PIRQA. Suppose it is connected to the IRQ16. This way, every time a processor has a signal that there is an interrupt on a IRQ16 line, it has to question all of the device drivers of PCI devices connected to that IRQ16 line (PIRQA) if they have an interrupt. If there is many of such devices, it surely wouldn't increase system response to the interrupt. And lanes PIRQB-PIRQD in this case would stand idle wasted most of the time. Here is a picture that illustrates this problem:<p/>
+	<p>As we've already said above, the most common case is when a PCI device has only one function, and its interrupt is connected to the INTA# lane (because why would hardware device developer route it in a different way?). Therefore if we decided to route all lanes as we've written, almost all of the devices in a system would share interrupt lane PIRQA. Suppose this lane is connected to the IRQ16. This way, every time a processor has a signal that there is an interrupt on a IRQ16 line, it has to question all of the device drivers of the PCI devices connected to that IRQ16 line (PIRQA) if they have an interrupt. If there are many of those devices, it surely wouldn't increase system response to the interrupt. And in this case lanes PIRQB-PIRQD would stand idle most of the time. Here is a picture that illustrates this problem:<p/>
```

Comment:
"And in this case lanes PIRQB-PIRQD would stand idle most of the time"
You've removed the 'wasted' keyword form this sentence. I guess it is not right for this situation, but can we write something to highlight a fact, that a valuable resource is not used in this case as it can be?

Reply:
"wasted" is ok, let's go back to that.